# GetActions.srv
string label
---
string[] actions

